libs/platform/src/lib/wizard-generator/components/wizard-generator/wizard-generator.component.html

```diff
@@ -1,16 +1,19 @@
-<ng-container *ngIf="_wizardCreated">
+<ng-container *ngIf="wizardCreated">
     <fdp-wizard-body
         [navigationButtonLabels]="navigationButtonLabels"
         [goNextButtonTemplate]="goNextButtonTemplate"
         [finishButtonTemplate]="finishButtonTemplate"
+        [summaryPageTemplate]="summaryPageTemplate"
         [responsivePaddings]="responsivePaddings"
-        [hidden]="!_wizardCreated"
+        [hidden]="!wizardCreated"
```

Comment:
what the sense is to have `[hidden]="!wizardCreated"` and parent with `*ngIf="wizardCreated"`?